### PR TITLE
[mapbox] potentially fail decode number

### PR DESCRIPTION
**Pull request by NuVivo314**
Solve incapacity to decode field if is a number

### CODE REVIEW

**Comment by codingsince1985** [comment]
Hi @NuVivo314,
Looks good to me.
Can you please add a test case as `response4` in [geocoder_test.go](https://github.com/NuVivo314/geo-golang/blob/634a3028093e390c27936f5bb204199d1c72fee6/mapbox/geocoder_test.go#L53) to cover this?

**Comment by NuVivo314** [comment]
@codingsince1985 test case update.

Thx

File: mapbox/geocoder_test.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ const (
                -37.75522
             ]
          },
-         "address":"60",
```
Comment (codingsince1985) [inline mapbox/geocoder_test.go]: `"address":"60"` is a real life response returned from mailbox, which shouldn't be changed.

File: mapbox/geocoder_test.go [hunk 2]
```diff
@@ -226,7 +226,7 @@ const (
                -37.738092
             ]
          },
-         "address":"60",
```
Comment (codingsince1985) [inline mapbox/geocoder_test.go]: same as above.

**Comment by codingsince1985** [comment]
Nice one!
Thank you @NuVivo314

**Pull request merged**